doc/faq.rst

```diff
@@ -110,7 +112,8 @@ in scikit-learn at least in some areas.
 Also note that your implementation need not be in scikit-learn to be used
 together with scikit-learn tools. You can implement your favorite algorithm in
 a scikit-learn compatible way, upload it to github and let us know. We will
-list it under :ref:`related_projects`.
+list it under :ref:`related_projects`, or even include it in
+`scikit-learn-contrib <http://scikit-learn-contrib.github.io>`_.
```

Comment:
I would write something like this:

```
You can implement your favorite algorithm in a scikit-learn compatible way,
upload it to github and let us know. We will very happily list it under
:ref:`related_projects`. If you already have a package on github following the
scikit-learn API, you may also be interested to look at `scikit-learn-contrib
<http://scikit-learn-contrib.github.io>`_.
```

The main thing is that IMO related projects and scikit-learn contrib looks too much alike in your original sentence and could potentially lead the reader to believe that we will maintain the project in scikit-learn-contrib which obviously is not the goal at all.